lib/utils.pm

```diff
@@ -902,6 +903,13 @@ sub install_all_from_repo {
     assert_script_run($exec_str);
 }
 
+sub gather_pidstat_data {
+    my ($pidstat_file) = @_;
+
+    script_run('pkill pidstat');
+    upload_logs($pidstat_file);
+}
+
```

Comment:
Maybe a util function which starts and closes any program in the background might be useful, just so we have a working way of doing it recorded, but I'm not that bothered either way. In fact the LTP compilation hasn't failed for a while so the bug we were asked to do this for might have been fixed. Otherwise this looks OK to me, I will try reusing it in the LTP test runner if we still need it.